.circleci/config.yml

```diff
@@ -6,18 +6,29 @@ jobs:
       - image: "cimg/elixir:1.13.3"
     steps:
       - checkout
+      - run: mix local.hex --force
```

Comment:
Do you know why we need to do this in CI but not locally?

Reply:
That is a good question, I put it in there to fix the errors I was getting after introducing credo as a dependency, but I did not really even think about until you mentioned it. After looking into it a bit, I think its because when I installed elixir locally with home brew, it installed hex too, but the docker image might not come with hex so we have to install it locally.

Reply:
Cool, thanks for clarifying that!